### PR TITLE
update design and state changes on appointment cards at Homepage Refrsh

### PR DESCRIPTION
**Pull request by jenniemc**
## Description
Make design and state changes based on  https://www.sketch.com/s/d4b8e6aa-2e36-47fd-aa04-52f4d3f839c3/a/JnPpm4M#Inspector

## Testing done
local and unit test

## Screenshots
Desktop
image

image

image

Mobile 
image

image

image

image

## Acceptance criteria
- [ ] Default state
- [ ] Hover state
- [ ]  Focus stage
- [ ]  Active state

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/appointment-list/components/AppointmentsPageV2/AppointmentListItem.jsx [hunk 1]
```diff
@@ -72,26 +73,30 @@ export default function AppointmentListItem({ appointment, facility }) {
...
-      className="vaos-appts__card vaos-appts__card--clickable"
+      className="vaos-appts__card--clickable vads-u-margin-bottom--3"
       data-cy="appointment-list-item"
     >
       <div
-        className="vads-u-padding--2 medium-screen:vads-u-padding--3 medium-screen:vads-u-margin-bottom--3 vads-u-display--flex vads-u-align-items--center"
+        className="vads-u-padding--2 vads-u-display--flex vads-u-align-items--left vads-u-flex-direction--column medium-screen:vads-u-padding--3 medium-screen:vads-u-flex-direction--row medium-screen:vads-u-align-items--center"
         onClick={() =>
-          !window.getSelection().toString() ? history.push(link) : null
+          !window.getSelection().toString()
```
Comment (jbalboni) [inline src/applications/vaos/appointment-list/components/AppointmentsPageV2/AppointmentListItem.jsx:88]: I wonder if this snippet is getting a little too clever? We could make it more obvious and in line with our normal style:

```
onClick={() => {
  if (!window.getSelection().toString()) {
    focusElement(`#${idClickable}`);
    history.push(link);
  }
}}
```

File: src/applications/vaos/appointment-list/components/AppointmentsPageV2/RequestListItem.jsx [hunk 2]
```diff
@@ -15,34 +16,38 @@ export default function RequestListItem({ appointment, facility }) {
...
-      className="vaos-appts__card vaos-appts__card--clickable"
+      className="vaos-appts__card--clickable vads-u-margin-bottom--3"
       data-cy="requested-appointment-list-item"
     >
       <div
-        className="vads-u-padding--2 medium-screen:vads-u-padding--3 medium-screen:vads-u-margin-bottom--3 vads-u-display--flex vads-u-align-items--center"
+        className="vads-u-padding--2 vads-u-display--flex vads-u-align-items--left vads-u-flex-direction--column medium-screen:vads-u-padding--3 medium-screen:vads-u-flex-direction--row medium-screen:vads-u-align-items--center"
         onClick={() =>
-          !window.getSelection().toString() ? history.push(link) : null
+          !window.getSelection().toString()
```
Comment (jbalboni) [inline src/applications/vaos/appointment-list/components/AppointmentsPageV2/RequestListItem.jsx:31]: Similar comment to above about this handler

File: src/applications/vaos/appointment-list/components/AppointmentsPageV2/AppointmentListItem.jsx [hunk 3]
```diff
@@ -72,26 +73,30 @@ export default function AppointmentListItem({ appointment, facility }) {
...
+            ? (focusElement(`#${idClickable}`), history.push(link))
```
Comment (jbalboni) [inline src/applications/vaos/appointment-list/components/AppointmentsPageV2/AppointmentListItem.jsx:89]: I could see a screen reader doing something weird with this extra focus, but VoiceOver seems fine with it, so hopefully the rest are as well.

**Review by jbalboni — APPROVED** [review]
One minor comment about the onClick handler, but otherwise everything looks good to me.

File: src/applications/vaos/appointment-list/components/AppointmentsPageV2/AppointmentListItem.jsx (hunk 1 above)
Comment (jenniemc) [reply src/applications/vaos/appointment-list/components/AppointmentsPageV2/AppointmentListItem.jsx]: Done

File: src/applications/vaos/appointment-list/components/AppointmentsPageV2/RequestListItem.jsx (hunk 2 above)
Comment (jenniemc) [reply src/applications/vaos/appointment-list/components/AppointmentsPageV2/RequestListItem.jsx]: Done

File: src/applications/vaos/appointment-list/components/AppointmentsPageV2/AppointmentListItem.jsx (hunk 3 above)
Comment (jenniemc) [reply src/applications/vaos/appointment-list/components/AppointmentsPageV2/AppointmentListItem.jsx]: Done

**Review by vbahinwillit — APPROVED** [review]

**Pull request merged**